atlasdb-backup/src/main/java/com/palantir/atlasdb/backup/AtlasRestoreService.java

```diff
@@ -112,7 +112,8 @@ public static AtlasRestoreService create(
 
     private void repairTransactionsTables(
```

Comment:
I meant refactor this method name, but it wasn't letting me comment on this line